### PR TITLE
23w45a Shinanigans

### PR DESCRIPTION
**Pull request by zOnlyKroks**
First contribution to yarn, just a random assortment of classes that came first under my mouse in enigma.

### CODE REVIEW

File: mappings/net/minecraft/client/render/debug/BreezeDebugRenderer.mapping [hunk 1]
```diff
@@ -0,0 +1,53 @@
+CLASS net/minecraft/class_8985 net/minecraft/client/render/debug/BreezeDebugRenderer
+	FIELD field_47465 PINK I
+	FIELD field_47466 LIGHT_BLUE I
+	FIELD field_47467 GREEN I
+	FIELD field_47468 ORANGE I
+	FIELD field_47469 RED I
+	FIELD field_47472 client Lnet/minecraft/class_310;
+	FIELD field_47473 breezeDebugInfoList Ljava/util/Map;
```
Comment (apple502j) [inline mappings/net/minecraft/client/render/debug/BreezeDebugRenderer.mapping]:
```suggestion
	FIELD field_47473 breezes Ljava/util/Map;
```

File: mappings/net/minecraft/client/render/debug/BreezeDebugRenderer.mapping [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+	METHOD <init> (Lnet/minecraft/class_310;)V
+		ARG 1 client
+	METHOD method_55255 clearMap ()V
```
Comment (apple502j) [inline mappings/net/minecraft/client/render/debug/BreezeDebugRenderer.mapping]:
```suggestion
	METHOD method_55255 clear ()V
```

Implementation detail should not be used as a name.

File: mappings/net/minecraft/client/render/debug/BreezeDebugRenderer.mapping [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+	METHOD method_55256 draw (ILorg/joml/Matrix4f;DDDLnet/minecraft/class_4588;Lnet/minecraft/class_243;FI)V
+		ARG 0 vertex
+		ARG 1 matrix4f
+		ARG 8 vertexConsumer
+		ARG 9 entityBox
+	METHOD method_55257 (Lnet/minecraft/class_1297;)Lnet/minecraft/class_243;
+		ARG 1 entity
+	METHOD method_55258 (Lnet/minecraft/class_8949;)Z
+		ARG 0 predicate
+	METHOD method_55259 renderOutlines (Lnet/minecraft/class_4587;Lnet/minecraft/class_4597;DDD)V
```
Comment (apple502j) [inline mappings/net/minecraft/client/render/debug/BreezeDebugRenderer.mapping]:
```suggestion
	METHOD method_55259 render (Lnet/minecraft/class_4587;Lnet/minecraft/class_4597;DDD)V
```

Mojang forgot to override `DebugRenderer.Renderer`; if they did, this method would've been named "render".

File: mappings/net/minecraft/client/render/debug/BreezeDebugRenderer.mapping [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+		ARG 1 stack
+		ARG 2 vertexConsumerProvider
+		ARG 3 x
+		ARG 5 y
+		ARG 7 z
```
Comment (apple502j) [inline mappings/net/minecraft/client/render/debug/BreezeDebugRenderer.mapping]:
```suggestion
		ARG 1 matrices
		ARG 2 vertexConsumers
		ARG 3 cameraX
		ARG 5 cameraY
		ARG 7 cameraZ
```

File: mappings/net/minecraft/client/render/debug/BreezeDebugRenderer.mapping [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+	METHOD method_55260 (Lnet/minecraft/class_4587;Lnet/minecraft/class_4597;DDDLnet/minecraft/class_8949;Lnet/minecraft/class_243;)V
+		ARG 9 entityVec3
```
Comment (apple502j) [inline mappings/net/minecraft/client/render/debug/BreezeDebugRenderer.mapping]:
```suggestion
		ARG 9 targetPos
```

File: mappings/net/minecraft/client/render/debug/BreezeDebugRenderer.mapping [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+	METHOD method_55261 (Lnet/minecraft/class_4587;Lnet/minecraft/class_4597;DDDLnet/minecraft/class_8949;Lnet/minecraft/class_2338;)V
+		ARG 9 targetVec4
+			COMMENT targetVec4
```
Comment (apple502j) [inline mappings/net/minecraft/client/render/debug/BreezeDebugRenderer.mapping]:
```suggestion
		ARG 9 jumpTarget
```

File: mappings/net/minecraft/client/render/debug/BreezeDebugRenderer.mapping [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+	METHOD method_55262 constructVertex (Lnet/minecraft/class_4587;Lnet/minecraft/class_4597;DDDLnet/minecraft/class_243;Lnet/minecraft/class_243;I)V
+		ARG 0 stack
+		ARG 1 vertexConsumerProvider
+		ARG 2 x
+		ARG 4 y
+		ARG 6 z
+		ARG 8 vec3
+		ARG 9 vec4
```
Comment (apple502j) [inline mappings/net/minecraft/client/render/debug/BreezeDebugRenderer.mapping]:
```suggestion
	METHOD method_55262 drawLine (Lnet/minecraft/class_4587;Lnet/minecraft/class_4597;DDDLnet/minecraft/class_243;Lnet/minecraft/class_243;I)V
		ARG 0 matrices
		ARG 1 vertexConsumers
		ARG 2 cameraX
		ARG 4 cameraY
		ARG 6 cameraZ
		ARG 8 entityPos
		ARG 9 targetPos
```

It's always a good idea to check other classes for naming conventions. Also, never name something "vec3" or "vec4". Vec4 is a vector of 4 values.

File: mappings/net/minecraft/client/render/debug/BreezeDebugRenderer.mapping [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+		ARG 10 color
+	METHOD method_55263 (Lnet/minecraft/class_746;Lnet/minecraft/class_4587;Lnet/minecraft/class_4597;DDDLnet/minecraft/class_8949;)V
+		ARG 10 breezeEntity
```
Comment (apple502j) [inline mappings/net/minecraft/client/render/debug/BreezeDebugRenderer.mapping]:
```suggestion
		ARG 10 breeze
```

File: mappings/net/minecraft/client/render/debug/BreezeDebugRenderer.mapping [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+	METHOD method_55264 (Lnet/minecraft/class_746;Ljava/lang/Integer;)Lnet/minecraft/class_1297;
+		ARG 1 breezeId
```
Comment (apple502j) [inline mappings/net/minecraft/client/render/debug/BreezeDebugRenderer.mapping]:
```suggestion
```

File: mappings/net/minecraft/client/render/debug/BreezeDebugRenderer.mapping [hunk 10]
```diff
@@ -0,0 +1,53 @@
...
+	METHOD method_55265 renderBox (Lorg/joml/Matrix4f;DDDLnet/minecraft/class_4588;Lnet/minecraft/class_243;FI)V
+		ARG 0 stack
+		ARG 1 x
+		ARG 3 y
+		ARG 5 z
+		ARG 7 vertexConsumerProvider
+		ARG 8 entityBox
```
Comment (apple502j) [inline mappings/net/minecraft/client/render/debug/BreezeDebugRenderer.mapping]:
```suggestion
```
FYI: This is not a box.

**Review by apple502j — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/client/render/debug/BreezeDebugRenderer.mapping [hunk 11]
```diff
@@ -0,0 +1,48 @@
...
+		ARG 1 matrices
+		ARG 2 vertexConsumers
+		ARG 3 cameraX
+		ARG 5 cameraY
+		ARG 7 cameraZ
+	METHOD method_55260 (Lnet/minecraft/class_4587;Lnet/minecraft/class_4597;DDDLnet/minecraft/class_8949;Lnet/minecraft/class_243;)V
+		ARG 9 targetPos
+	METHOD method_55261 (Lnet/minecraft/class_4587;Lnet/minecraft/class_4597;DDDLnet/minecraft/class_8949;Lnet/minecraft/class_2338;)V
+		ARG 9 targetVec4
+			COMMENT jumpTarget
```
Comment (Shnupbups) [inline mappings/net/minecraft/client/render/debug/BreezeDebugRenderer.mapping]:
```suggestion
		ARG 9 jumpTarget
```

Comment (zOnlyKroks) [reply mappings/net/minecraft/client/render/debug/BreezeDebugRenderer.mapping]: huh did i not just fix those

**Review by apple502j — APPROVED** [review]

**Pull request merged**